packages/test-setup/src/index.ts

```diff
@@ -1,5 +1,8 @@
 import type { Options } from '@embroider/compat';
 
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const currentEmbroiderVersion = require('../package.json').version;
```

Comment:
Note we *could* use `import` here, but that would require to add `"resolveJsonModule": true` to the global tsconfig, so was not sure if you would want that.